tasks/need_fixing_linters/need_fixing_linters.py

```diff
@@ -0,0 +1,58 @@
+"""
+Invoke tasks to fix the linter
+"""
+
+from invoke import task
+from invoke.exceptions import Exit
+from ..libs.pipeline_notifications import GITHUB_SLACK_MAP
+from .golangci_lint_parser import filter_lints, display_result, merge_results
+
+FIRST_COMMIT_HASH = "52a313fe7f5e8e16d487bc5dc770038bc234608b"
+# See https://go.dev/doc/install/source#environment for all available combinations of GOOS x GOARCH.
+CI_TESTED_OS_AND_ARCH = {"linux": ["amd64", "arm64"], "windows": ["amd64"], "darwin": ["amd64", "arm64"]}
+
+def check_if_team_exists(team: str):
+    """
+    Check if an input team exists in the GITHUB_SLACK_MAP. Exits the code if it doesn't.
+    """
+    if team:
+        team = team.lower()
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I think if you accept uppercased team you'll have a mismatch when comparing it to the value returned by `codeowners` (it will return a lowercase value)
You should either lowercase the team variable in `need_fixing_linters` or fail if the casing is not exactly the good one

Reply:
and you can use `casefold` ([link](https://docs.python.org/3/library/stdtypes.html?highlight=casefold#str.casefold)) rather than `lower` in any case

Reply:
Right thank you both ! This is fixed now. I kept using the `lower` method though since I think it's more readable and that the team names won't contain weird chars.